test/integration/default/inspec.yml

```
@@ -1,5 +1,5 @@
 name: template
-title: Template Formula
+title: template Formula
```

Let's remove all capitalisation here for consistency with the `FORMULA` file:

```suggestion
title: template formula
```